platforms/ios/lib/WysiwygComposer/Sources/WysiwygComposer/Components/WysiwygComposerView/WysiwygTextView.swift

```diff
@@ -172,6 +202,10 @@ public class WysiwygTextView: UITextView {
 
         wysiwygDelegate.textView(self, didReceivePasteWith: provider)
     }
+    
+    deinit {
+        NotificationCenter.default.removeObserver(self)
```

Comment:
I don't think this is needed

Reply:
yeah the stack overflow solution used vert old code, now these are deallocated automatically